Hoxtygen/Library-react
# [#169110826] : Add postgresql database

# What does this PR do
Adds PostgreSQL database to the project for data persistence.

## Description of the task to be completed
Have the application persist data by configuring it to use PostgreSQL, a relational database.
 
## How should this manually be tested
- clone the repo
- install PostgreSQL on your local machine
- create a .env file using the `.env.sample` as a template
- open a terminal, cd into the project folder and run `npm install`
- run `npx knex migrate:latest`
- run `npx knex seed:run`
- run `select * from users;` in your database to see the seeded data.

## What are the relevant pivotal tracker stories
[[#169110826](https://www.pivotaltracker.com/story/show/169110826)]
